### ISSUE TITLE
why during validation not using model.eval()?

### ISSUE THREAD

**Issue by Ponytai1j** [nshaud/DeepHyperX#51]
Since some model have BN,dropout layers, why dont use model.eval() in the val function?   

am really confused, hope someone can help me. thx in advance.

**Comment by Neymar1011** [comment]
Hi , maybe the val function do not effect the training result.
but i think it's necessary to use model.eval() before validation.

**Comment by Neymar1011** [comment]
Hi , maybe the val function do not effect the training result.
but i think it's necesseray to use model.eval() before validation.
